reconcile/queries.py

```diff
@@ -509,3 +509,21 @@ def get_saas_files():
     """ Returns SaasFile resources defined in app-interface """
     gqlapi = gql.get_api()
     return gqlapi.query(SAAS_FILES_QUERY)['saas_files']
+
+
+GITLAB_OWNERS_QUERY = """
+{
+  apps: apps_v1 {
+    codeComponents {
+      url
+      gitlabOwners
+    }
+  }
+}
+"""
+
+
+def get_apps_gitlab_owners_repos():
+    """ Returns all codeComponents. """
+    gqlapi = gql.get_api()
+    return gqlapi.query(GITLAB_OWNERS_QUERY)['apps']
```

Comment:
can we process this and return the list of repos?
take a look at `get_repos` in this file (using `server` will probably also be good here)

Reply:
indeed. done!